osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -34,7 +34,11 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
 
             double aimRating = Math.Sqrt(skills[0].DifficultyValue()) * difficulty_multiplier;
             double speedRating = Math.Sqrt(skills[1].DifficultyValue()) * difficulty_multiplier;
-            double starRating = aimRating + speedRating + Math.Abs(aimRating - speedRating) / 2;
+            
+            double baseAimPerformance = Math.Pow(5 * Math.Max(1, aimRating) - 4, 3) / 100000;
+            double baseSpeedPerformance = Math.Pow(5 * Math.Max(1, speedRating) - 4, 3) / 100000;
```

Comment:
These are missing the `0.0675` divisor mentioned in the docs. Due to that almost everything is no higher than 1* on a quick check.

Should presumably be

```suggestion
            double baseAimPerformance = Math.Pow(5 * Math.Max(1, aimRating / 0.0675) - 4, 3) / 100000;
            double baseSpeedPerformance = Math.Pow(5 * Math.Max(1, speedRating / 0.0675) - 4, 3) / 100000;
```

But it may be a better idea yet to *not* duplicate this formula in two places, and instead store the result of the above in the difficulty attributes and read that directly in performance point calculations.